cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/query/IndexUpdater.java

```diff
@@ -390,4 +319,100 @@ public DBParameter(String tableName, ContentValues contentValues) {
         }
     }
 
+    private static class SequenceNumberForIndexCallable implements SQLCallable<Long> {
```

Comment:
Yes they should, but that would be part of the issue to host them into callables and consolidate where required / appropriate  .